routing/pedestrian_directions.cpp

```diff
@@ -35,6 +35,7 @@ PedestrianDirectionsEngine::PedestrianDirectionsEngine()
 void PedestrianDirectionsEngine::Generate(IRoadGraph const & graph, vector<Junction> const & path,
                                           Route::TTimes & times, Route::TTurns & turns,
                                           vector<Junction> & routeGeometry,
+                                          vector<traffic::TrafficInfo::RoadSegmentId> & /* routeSegs */,
```

Comment:
Yes. If to look for routeSegs there're a lot which should be called trafficSegs . I'll rename them.